source/include/boot_upgrade_package.h

```diff
@@ -6,13 +6,17 @@
 
 typedef struct
```

Comment:
That's absolute fine. Could you just add some comments about the structure and its fields and what are they for?

```
/*
 * Upgrade Package Metadata
 */
typedef struct
{
    /* Upgrade Package Metadata Prefix : Must be ZAYA */
    uint8_t prefix[4];
    /* Version of the image/application */
    uint32_t version;
    /* Image size */
    uint32_t size;
    uint8_t signature[64];
} boot_image_metadata_t;
```